packages/contracts-sdk/src/IncentivesFundManager.types.ts

```
@@ -0,0 +1,38 @@
+  };
+} | {
+  native_token: {
+    denom: string;
+  };
+};
+export interface Asset {
+  amount: Uint128;
+  info: AssetInfo;
+}
+export type QueryMsg = {
+  config: {};
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)